lib/web_ui/test/frame_timings_common.dart

```diff
@@ -0,0 +1,46 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// @dart = 2.6
+import 'dart:async';
+
+import 'package:test/test.dart';
+import 'package:ui/ui.dart' as ui;
+
+/// Tests frame timings in a renderer-agnostic way.
+///
+/// See CanvasKit-specific and HTML-specific test files `frame_timings_test.dart`.
+Future<void> runFrameTimingsTest() async {
+  List<ui.FrameTiming> timings;
+  ui.window.onReportTimings = (List<ui.FrameTiming> data) {
+    timings = data;
+  };
+  Completer<void> frameDone = Completer<void>();
+  ui.window.onDrawFrame = () {
+    final ui.SceneBuilder sceneBuilder = ui.SceneBuilder();
+    sceneBuilder
+      ..pushOffset(0, 0)
+      ..pop();
+    ui.window.render(sceneBuilder.build());
+    frameDone.complete();
+  };
+
+  // Frame 1
```

Comment:
nit: I have told all comments have to end with a '.'  (same for frame 2)

Reply:
Done.